### PR TITLE
Merge ITEM.THING and ITEM.TypeThing translations

### PR DESCRIPTION
**Pull request by giant-teapot**
Remove all `ITEM.THING` translation strings in favor of `ITEM.TypeThing`.
The former were only used to display item types in the chat.

`ITEM.*` elements were also sorted alphabetically to make the merging
process easier.

This changes has the benefit of adding many already-existing
translations for item types.
However, the "new" item types do not yet have translations for all
languages:

 * `ITEM.TypeArmor`
 * `ITEM.TypeMonsterattack`
 * `ITEM.TypeMonstertalent`
 * `ITEM.Weapon`

### CODE REVIEW

**Comment by aMediocreDad** [comment]
I agree that there is a lot of potential overlap in the future as we expand localisation in the system. 

As I was working on #30 last night I found I reused strings that in context had poor naming. I was thinking of doing a refactor of the strings we use renaming those that might fit a lot of usecases like common language, or systemwide item names. Maybe bringing these under a `COMMON` umbrella would be better. Of course this presents the argument that everything could fall under this umbrella, but I think being judicious in its use could save us some hassle later.

File: lang/cs.json [hunk 1]
```diff
@@ -113,15 +113,6 @@
   "WEAPON.SKILL_BONUS": "Dovednostní bonus",
   "WEAPON.FEATURE": "Vlastnosti",
 
-  "ITEM.ATTACK": "Útok",
```
Comment (aMediocreDad) [inline lang/cs.json:116]: Careful of deleting this one. There is no equivalent in TypeThing

Comment (giant-teapot) [reply lang/cs.json:116]: Actually, it was used for the monster attack. See `chat/item.html` line 23. :)

**Comment by aMediocreDad** [comment]
> However, the "new" item types do not yet have translations for all
> languages:
> 
> * `ITEM.TypeArmor`
> * `ITEM.TypeMonsterattack`
> * `ITEM.TypeMonstertalent`
> * `ITEM.Weapon`

This is a good catch. I did not see the already existing ones. And yes, merging existing ITEM.* into ITEM.TypeThing is a warranted change.

**Comment by giant-teapot** [comment]
I just hope it helps. If you like it, feel free to integrate it to #30, but otherwise I don't mind if you'd want to do it "your way". :)
It just seemed a more efficient (and potentially useful) way to convey the idea.

File: lang/cs.json (hunk 1 above)
Comment (aMediocreDad) [reply lang/cs.json:116]: > Actually, it was used for the monster attack. See `chat/item.html` line 23. :)
> Needless to say `Monsterattack` is MUCH improved naming-wise.

Haven't checked. This is the only use of this string?

Comment (giant-teapot) [reply lang/cs.json:116]: Yes, it was the only place it was used.
After this change, there are no remaining uses of `ITEM.ATTACK`.

**Pull request merged**